project/MleapProject.scala

```diff
@@ -95,6 +101,21 @@ object MleapProject {
     dependencies = Seq(runtime)
   )
 
+  lazy val xgboost = Project(
+    id = "mleap-xgboost",
+    base = file("mleap-xgboost"),
+    dependencies = Seq(runtime)
+  )
+
+  lazy val xgboostSpark = Project(
```

Comment:
Do you think we should include running these tests in travis.sh similarly to how we do the mleap-serving ones?

Reply:
It would be nice to run these in travis... Do you know of an easy way to include the xgboost shared library so we can run it in travis?

Reply:
I'll take a look, but we can do this even after the PR gets merged.